app/src/main/res/layout/fragment_article_edit_details.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_marginStart="16dp"
+    android:layout_marginEnd="16dp"
```

Comment:
These should be removed and add `android:layout_marginHorizontal="16dp"` to the child `LinearLayout` at line `11`. The current layout has extra horizontal paddings and the scrollbar is not sticking to the right edge.